src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/GoalService.kt

```diff
@@ -49,6 +50,8 @@ class GoalService(
       title = goal.title,
       areaOfNeed = areaOfNeedEntity,
       targetDate = goal.targetDate,
+      goalStatus = if (goal.targetDate != null) GoalStatus.ACTIVE else GoalStatus.FUTURE,
```

Comment:
could we just call this `status`? I don't we need to prefix `goal` onto the front, considering it's got the context of being on the goal entity